tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/IntegrationTestDatabase.cs

```diff
@@ -14,6 +14,7 @@ public static async Task ResetAsync(IConfiguration config)
             await databaseConnection.ExecuteAsync("GRANT CONNECT TO [NHSD-ORDAPI];");
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datareader ADD MEMBER [NHSD-ORDAPI];");
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datawriter ADD MEMBER [NHSD-ORDAPI];");
+            await databaseConnection.ExecuteAsync("DELETE FROM dbo.OrderItem;");
```

Comment:
Couldnt you use DELETE ON CASCADE, then you wont need this

Reply:
I will look into this, good shout 👍